jobs/update-colin-filings/k8s/cronjob.param

```diff
@@ -0,0 +1,11 @@
+#=========================================================
+# Template parameters for:
+# Project: colin-updater
+# Component: cronjob
+# Action: build
+# Template File: templates/cronjob.json
+#=========================================================
+NAME="update-colin-filings"
+NAMESPACE="gl2uos"
+ENV="dev"
```

Comment:
Again, dev or test? How do we know?

Reply:
Yeah the params are just placeholders (they are meant to be generic so you'd use thee same one for all envs and just change the values as you needed them)